Electron/src/RED/components/Map.tsx

```diff
@@ -96,14 +100,6 @@ class Map extends Component<IProps, IState> {
               zoom={this.state.zoom}
               maxZoom={this.state.maxZoom}
               id="map"
-              whenReady={(map: any): void =>
-                map.target.on("click", (e: { latlng: { lat: number; lng: number } }) => {
-                  this.props.store(new Date().toLocaleTimeString(), {
-                    lat: e.latlng.lat,
-                    lon: e.latlng.lng,
-                  })
-                })
-              }
```

Comment:
Potentially restore this and uncomment the componentDidMount component - it certainly doesn't seem to be hurting anything even if it isn't the full fix we are looking for for offline maps